packages/fast-components-react-base/.storybook/config.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { configure, addParameters, addDecorator } from "@storybook/react";
+import { create, Global } from "@storybook/theming";
+import theme from "@microsoft/fast-storybook-presets/dist/theme";
```

Comment:
should we export this from a root level instead of reaching into dist?

Reply:
added a comment - stuff blows up if we do it the other way for some reason.